src/instrumentation/fetch.ts

```diff
@@ -89,6 +99,15 @@ export function getParentContextFromHeaders(headers: Headers): Context {
 	})
 }
 
+function getParentContextFromRequest(request: Request) {
+	const workerConfig = getActiveConfig()
+	const acceptTraceContext =
+		typeof workerConfig.handlers.fetch.acceptTraceContext === 'function'
+			? workerConfig.handlers.fetch.acceptTraceContext(request)
```

Comment:
I am normally *extremely* against nested ternary expressions, but I am trying to hate this one without any luck :D

Reply:
Yeah it's not my favorite either, but it was concise 😅 It seemed slightly better than splitting this out and introducing some indirection in an already small function.